nearcore/src/lib.rs

```diff
@@ -264,11 +264,11 @@ pub fn start_with_config_and_synchronization(
     );
     client_adapter_for_shards_manager.bind(client_actor.clone().with_auto_span_context());
     let (shards_manager_actor, shards_manager_arbiter_handle) = start_shards_manager(
-        runtime.clone(),
+        view_runtime,
```

Comment:
Why should we use the view_runtime here?

Reply:
Thought we are accessing some garbage collectable data through runtime adapter there, but apparently not.